src/index.ts

```diff
@@ -79,26 +80,32 @@ export type SagaPromiseActionCreator<V, P, T extends string, TA extends PayloadA
 
 export type SagaPromisePreparedActionCreator<V, T extends string, TA extends PrepareAction<any>> = SagaPromiseActionCreator<V, ReturnType<TA>["payload"], T, _ActionCreatorWithPreparedPayload<TA, T>>;
 
+let promiseResolutionMap: PromiseResolutionMap<any> | undefined;
+
+function getPromiseResolution<V>(action: SagaPromiseAction<any, any, any>): PromiseResolution<V> | undefined {
+  return promiseResolutionMap?.get(action.type);
+}
+
 function isTriggerAction(action: SagaPromiseAction<any, any, any>) {
-  return action?.meta?.promiseActions.resolved != null;
+  return action?.meta?.actionSymbol === promiseActionSymbol;
```

Comment:
Here you would have to check the symbol to exist as property key and optionally for truthyness, since we assigned `true`.

Reply:
@teneko fixed